sales_team_security/tests/common.py

```diff
@@ -61,7 +68,7 @@ def _check_whole_permission_set(self, extra_checks=True):
         self._check_permission(self.user2, False, False)
         self._check_permission(False, self.team, True)
         if extra_checks:
-            self._check_permission(False, self.team2, False)
+            self._check_permission(False, self.team2, True)
```

Comment:
Changing this means that the module is not working as expected.

Reply:
This still doesn't convince me, as changing this means that the tests are changing the expected behavior. Can you explain why this change?

Reply:
@cesar-tecnativa Could you answer this ?